src/browser/createAlgoliasearch.js

```diff
@@ -87,6 +87,17 @@ module.exports = function createAlgoliasearch(AlgoliaSearch, uaSuffix) {
       // breaks it and set it to false by default
       if (req instanceof XMLHttpRequest) {
         req.open(opts.method, url, true);
+
+        // note: this breaks everything else but makes the browser tests pass
+        // the analytics endpoint _needs_ it via header, and doesn't accept QS
+        req.setRequestHeader(
+          'x-algolia-application-id',
+          opts.headers['x-algolia-application-id']
+        );
+        req.setRequestHeader(
+          'x-algolia-api-key',
+          opts.headers['x-algolia-api-key']
+        );
```

Comment:
I added this manually as request header for analytics. I guess it needs to be an option since without setting them explicitly as header (default is query string) I get a 401 error by the analytics API.

This either needs to be an option, either `inlineHeaders` needs to be an option, either the analytics API would accept API key / app id to be sent via query string. 

Last option is my preference